### ISSUE TITLE
bf-cache always fails in headless mode

### ISSUE THREAD

**Issue by adamraine** [GoogleChrome/lighthouse#14784]
With `--headless`:
![Screenshot 2023-02-10 at 9 02 53 AM](https://user-images.githubusercontent.com/6752989/218152004-d5c7d9ff-3f76-4709-9118-1015efe7991a.png)

**Comment by dwsmart** [comment]
Running in the new headless mode seems to fix this, so changing

```
const chrome = await chromeLauncher.launch({chromeFlags: ['--headless']});
```

to

```
const chrome = await chromeLauncher.launch({chromeFlags: ['--headless=new']});
```

Seems to solve the `window.open()` issue:
![screenshot of lighthouse 10.0.1 test for https://example,com, using new headless mode](https://user-images.githubusercontent.com/11179452/220645617-8c446abe-a1a0-460e-b388-d6b549f6d90f.png)

**Comment by adamraine** [comment]
I doubt the old headless Chrome will every support bfcache testing, so we shouldn't consider this a bug. That being said, we should warn the user when the results are foobar and/or mark the audit as n/a.

**Comment by Yogurt4** [comment]
> That being said, we should warn the user when the results are foobar and/or mark the audit as n/a.

Agreed. Lighthouse gives me a "Page prevented back/forward cache restoration" fail. Testing it with Chrome DevTools gives "Successfully served from back/forward cache." though.
